lib/components/file-manager.js

```diff
@@ -55,41 +55,37 @@ var fileManager = {
 					return console.error(err);
 				}
 
-				this.watched(function (err, watched) {
-					if (err) {
-						return console.error(err);
-					}
-					_.forEach(_.filter(_.flatten(_.values(watched)), function (file) {
-						return path.extname(file);
+				var watched = this.watched();
+				_.forEach(_.filter(_.flatten(_.values(watched)), function (file) {
```

Comment:
```
_.chain(watched).values()
   .flatten()
   .filter(function (file) {
       return path.extname(file);
    })
   .forEach(function(file) {
        var cat = getCategory(file);
        cache[cat.name].filePaths.push(file);
   }).commit();
```
The .commit() is essential because the chain will only be executed on a .commit() or .value()
I will commit it on the branch